blog-migration.js

```diff
@@ -36,14 +37,31 @@ const MIGRATION = {
     'banner': 'banner',
 }
 
+async function updateSave(outputDocxFile, cached, mdast, sourceDocxFile, force, report, entry) {
+    console.log(`Saving ${outputDocxFile}`);
+    if (cached) {
+        try {
+            await saveUpdatedDocx(mdast, sourceDocxFile, outputDocxFile, force);
+        } catch (e) {
+            console.warn(`Error updating ${sourceDocxFile}`, e.message);
+            report.warned.push({ entry, message: e.message });
+            await saveDocx(mdast, outputDocxFile, force);
+        }
+    } else {
+        await saveDocx(mdast, outputDocxFile, force);
+    }
+}
+
 export async function main(index, cached, output, force) {
     const reportDir = `${REPORT_DIR}/${PROJECT}`;
     const mdDir = `${MD_DIR}/${PROJECT}`;
     const docxDir = `${DOCX_DIR}/${PROJECT}`;
     const outputDir = `${output}/${PROJECT}`;
     const report = { succeed: [], skipped: [], failed: [], warned: [] };
-    const indexUrl = `${SITE}${index}`;
-    const entries = await loadIndex(PROJECT, indexUrl, cached);
+    // const entries = await loadIndex(PROJECT, `${SITE}${index}`, cached);
+    const top100 = await readIndex('bacom-blog/top-100.json');
+    const banners = await readIndex('bacom-blog/banner.json');
+    const entries = [...top100, ...banners];
```

Comment:
Shouldn't this get put back to entries as line 61?

Reply:
How does a conditional sound?

Reply:
You mean like, if test run the top100 else run all? Honestly, whatever is fine. I guess we have a cleanup ticket so it'll be changing anyway